counter/Counter.js

```diff
@@ -1,7 +1,14 @@
+let id = 1;
+
 class Counter {
 	constructor(parent) {
+		this.DEFAULT_VALUE = 0;
+		this.MAX_VALUE = 3;
+
+		this.id = id++;
```

Comment:
Обычно в конструкторе инициализируем те переменные, которые экземпляр-специфик: мы при создании класса передаём какую-то инфу в аргументах и в зависимости от этих значений создаём экземпляр класса. Когда у тебя переменные нужны для внутреннего использования и они не зависят от аргументов, переданных при создании экземпляра, можно сохранять их вне конструктора
```suggestion
class Counter {
       DEFAULT_VALUE = 0;
       MAX_VALUE = 3;
       id = id++;
       
	constructor(parent) {
```